addons/chemical/functions/fnc_handleCSGas.sqf

```diff
@@ -18,6 +18,9 @@ params ["_logic","_radius"];
 [{
     params["_params","_handler"];
     _params params["_unit"];
+    if (!isDamageAllowed _unit) exitWith {
+        [_unit] call FUNC(clearChemicalInjuriesLocal);    
+    };
```

Comment:
empty line after if statement to increase readability.

Reply:
addresed